TestApps/TestApp_Swift/AppDelegate.swift

```diff
@@ -0,0 +1,58 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import UIKit
+import AEPCore
+import AEPServices
+import AEPLifecycle
+import AEPSignal
+import AEPIdentity
+import AEPAssurance
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    private let LAUNCH_ENVIRONMENT_FILE_ID = ""
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        let appState = application.applicationState
+        let extensions = [AEPIdentity.Identity.self,
+                          Lifecycle.self,
+                          Signal.self,
+                          Assurance.self]
+
+        MobileCore.registerExtensions(extensions, {
+            MobileCore.configureWith(appId: self.LAUNCH_ENVIRONMENT_FILE_ID)
+            if appState != .background {
```

Comment:
We can remove this logic and allow full control of start and pause APIs in the Lifecycle view.